src/autoComplete/index.unit.tsx

```diff
@@ -343,18 +343,18 @@ describe('AutoComplete', () => {
 
   describe('#focus', () => {
     it('Can add focus to TextField', () => {
-      const wrapper = shallow(<AutoComplete {...defaultProps} focus />)
+      const wrapper = mount(<AutoComplete {...defaultProps} focus />)
       expect(wrapper.find('TextField').prop('focus')).toBe(true)
     })
     it('Should not add border focus on Textfield', () => {
-      const wrapper = shallow(<AutoComplete {...defaultProps} />)
+      const wrapper = mount(<AutoComplete {...defaultProps} />)
```

Comment:
Why `mount` ?

Reply:
Because wrapper.find('TextField') can only work with mount since now it's wrapped in the Styled HOC